labelbox/data/annotation_types/data/tiled_image.py

```diff
@@ -0,0 +1,364 @@
+import math
+import logging
+from enum import Enum
+from typing import Optional, List, Tuple
+from concurrent.futures import ThreadPoolExecutor
+
+import requests
+import numpy as np
+import tensorflow as tf
+from retry import retry
+from pyproj import Transformer
+from pydantic import BaseModel, validator, conlist
+from pydantic.class_validators import root_validator
+
+from ..geometry import Point
+from .base_data import BaseData
+from .raster import RasterData
+"""TODO: consider how to swap lat,lng to lng,lt when version = 2...
```

Comment:
@msokoloff1 not sure how to approach this one...originally left validator inside the TiledBounds class since it made the most sense to me, but if a user inputs lng,lt instead and has version=2 in the old import methodology, it would be okay because it flipped

Reply:
The cool part about annotation types is we can just set a specific format that we expect. It is the converters problem to deal with mapping different formats into the annotation type spec.